### PR TITLE
trace-dispatcher: optional EKG.Store.

### PR DESCRIPTION
**Pull request by denisshevchenko**
This PR makes `EKG.Store` optional for `initForwarding`. Uses updated version of `ekg-forward` library.

### CODE REVIEW

File: cardano-node/src/Cardano/Node/Tracing/API.hs [hunk 1]
```diff
@@ -96,7 +96,7 @@ initTraceDispatcher nc p networkMagic nodeKernel p2pMode = do
           -- TODO: check if this is the correct way to use withIOManager
           (forwardSink, dpStore) <- withIOManager $ \iomgr -> do
             let tracerSocketMode = Just . first unSocketPath =<< ncTraceForwardSocket nc
-            initForwarding iomgr trConfig networkMagic ekgStore tracerSocketMode
+            initForwarding iomgr trConfig networkMagic (Just ekgStore) {- FIXME: for compile -} tracerSocketMode
```
Comment (denisshevchenko) [inline cardano-node/src/Cardano/Node/Tracing/API.hs:99]: This should be changed: we have to add the case when `ekgStore` is `Nothing`.

**Review by deepfire — APPROVED** [review]
Looks fine to me, thank you @denisshevchenko

**Review by Jimbo4350 — APPROVED** [review]

**Pull request merged**